pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, fetchpatch
+, dbus_glib, gtk2, gtk3, libexif, libpulseaudio, libXScrnSaver, ninja, nss
+, pciutils, pkgconfig, python2, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "vivaldi-ffmpeg-codecs";
+  version = "59.0.3071.104";
```

Comment:
By out of sync I was talking about the case when a maintainer will forget about it and confusion will result. Any solution (for example adding a maintainer script) that solves that is what I would like to see. 

A secondary goal is efficiency of the amount of CPU being used.